VirtoCommerce.CustomerModule.Data/Search/Indexing/MemberDocumentBuilder.cs

```diff
@@ -162,5 +168,30 @@ protected virtual void AddParentOrganizations(IndexDocument document, ICollectio
             document.AddFilterableValues("ParentOrganizations", nonEmptyValues);
             document.AddFilterableValue("HasParentOrganizations", nonEmptyValues?.Any() ?? false);
         }
+
+        protected virtual void IndexDynamicProperty(IndexDocument document, DynamicObjectProperty property)
+        {
+            var propertyName = property.Name?.ToLowerInvariant();
+            if (!string.IsNullOrEmpty(propertyName))
+            {
+                var isCollection = property.IsDictionary || property.IsArray;
+                var values = new List<object>();
+
+                if (property.IsDictionary)
+                {
+                    //add all locales in dictionary to searchIndex
+                    values.AddRange(property.Values.Select(x => x.Value)
+                                                    .Cast<DynamicPropertyDictionaryItem>()
+                                                    .Where(x => !string.IsNullOrEmpty(x.Name))
+                                                    .Select(x => x.Name));
+                }
+                else
+                {
+                    values.AddRange(property.Values.Where(x => x.Value != null)
+                                                    .Select(x => x.Value));
+                }
+                document.Add(new IndexDocumentField(propertyName, values) { IsRetrievable = true, IsFilterable = true, IsCollection = isCollection });
```

Comment:
Add empty line before this one

Reply:
![image](https://user-images.githubusercontent.com/2816674/49963558-fb014180-ff20-11e8-935f-c10dbe47773a.png)

Reply:
It was a different code (switch/case block)